### PR TITLE
AEH/udp_filter

### PR DESCRIPTION
**Pull request by aidan-hubley**
**Changes**
- Added Add Circle button to the Filter component
- Added All Circle Modal button to the Filter component

**Notes**
- Circles in the All Circle Modal have their title, maybe the title should be present in the carousel as well? @dev-arekusandoru 

**YouTrack Link**
[PC-122](https://prayercircle.youtrack.cloud/agiles/147-2/current?issue=PC-122)

### CODE REVIEW

**Comment by dev-arekusandoru** [comment]
@aidan-hubley I think the circle name over the circle like you had in your branch is a cool idea. Like i mentioned in slack i think [react-native-bottom-sheet](https://github.com/gorhom/react-native-bottom-sheet) is a better approach to display all circles. also as mentioned in slack the repositioning of the buttons will help maintain our streamlined design.

**Comment by aidan-hubley** [comment]
@dev-arekusandoru I am looking to complete my time on this branch. Feel free to implement the title effect on the carousel and/or pick up where I left off with the bottom the new BottomSheetModal. I am struggling and don't think it's fair (or in line with MVP) to implement new libraries mid development. I have sunk an extra 3 hours in struggling with the new library. It's all yours @dev-arekusandoru

**Review by dev-arekusandoru — APPROVED** [review]
Sure, I'll add the new bottom sheet. I don't think it is out of line with the MVP to include this library to achieve a certain outcome. There have been many instances where different libraries have been added mid development for the purpose of a more complete or finished looking piece.

File: components/Filter.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ const Filter = forwardRef((props, ref) => {
 			/>
 			<AnimatedView
 				style={opacityStyle}
-				className='absolute w-screen h-[200px] max-w-[500px] flex items-start justify-center overflow-visible'
+				className='absolute w-screen h-[250px] max-w-[500px] flex items-start justify-center overflow-visible'
```
Comment (dev-arekusandoru) [inline components/Filter.js]:
```suggestion
				className='absolute w-screen h-[200px] max-w-[500px] flex items-start justify-center overflow-visible'
```
revert back to original height

**Pull request merged**